go/logic/applier.go

```diff
@@ -974,6 +1013,15 @@ func (this *Applier) AtomicCutOverMagicLock(sessionIdChan chan int64, tableLocke
 		return err
 	}
 
+	if this.migrationContext.CutOverIdleTimeoutSeconds >= 1 {
+		restoreIdleTimeoutFunc, err := this.InitAtomicCutOverIdleTimeout(tx)
```

Comment:
IMO should be replaced with:
```suggestion
		defer func() {
		if _, err := sqlutils.ExecNoPrepare(this.db, query); err != nil {
 			this.migrationContext.Log.Errorf("Failed to restore applier wait_timeout to %d seconds: %v",
 				this.migrationContext.ApplierWaitTimeout, err,
 			)
 		}
 		}
		err := this.InitAtomicCutOverIdleTimeout(tx)
```

Reply:
@shlomi-noach makes sense 👍. I added a few questions to your responses; I'll make this change once we've reached agreement. Thanks for checking 🙇

Reply:
Updated 👍